src/sentry/static/sentry/app/components/activity/noteInput.jsx

```diff
@@ -206,6 +231,21 @@ const NoteInput = React.createClass({
     }
   },
 
+  getMemberData() {
+    let sessionUser = ConfigStore.get('user');
+    let results = [];
+    this.state.memberList.forEach(member => {
```

Comment:
does this data change ever? if it doesn't and if you always need `memberList` in this format, then i'd just do this work in `getInitialState` rather than calling this repeatedly from `render`